e2e-tests/auto-tuning/run

```diff
@@ -7,56 +7,87 @@ test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
 cluster="auto-tuning"
 
+does_autotune_cm_exists() {
+	set +e
+	kubectl_bin get configmap auto-${cluster}-pxc
+	set -e
+}
+
 get_variable_from_cm() {
 	kubectl_bin get configmap auto-${cluster}-pxc -o yaml | grep -oP "$1 = \K[0-9]+"
 }
 
 create_infra $namespace
 
-spinup_pxc "$cluster" "${test_dir}/conf/${cluster}1.yml"
+spinup_pxc "$cluster" "${test_dir}/conf/${cluster}-with-limits.yml"
 
-AUTO_INNODB_SIZE=$(run_mysql \
+INNODB_SIZE=$(run_mysql \
 	'SELECT @@innodb_buffer_pool_size;' \
 	"-h $cluster-pxc -uroot -proot_password")
-AUTO_CONNECTIONS=$(run_mysql \
+CONNECTIONS=$(run_mysql \
 	'SELECT @@max_connections;' \
 	"-h $cluster-pxc -uroot -proot_password")
 
 cm_buffer_size=$(get_variable_from_cm innodb_buffer_pool_size)
-if [[ ${AUTO_INNODB_SIZE} != ${cm_buffer_size} ]]; then
-	echo "innodb_buffer_pool_size is set to ${AUTO_INNODB_SIZE}, which does not correlate with configmap value: ${cm_buffer_size}"
+if [[ ${INNODB_SIZE} != ${cm_buffer_size} ]]; then
+	echo "with-limits: innodb_buffer_pool_size is set to ${INNODB_SIZE}, which does not correlate with configmap value: ${cm_buffer_size}"
 	exit 1
 fi
 
 cm_max_connections=$(get_variable_from_cm max_connections)
-if [[ ${AUTO_CONNECTIONS} != ${cm_max_connections} ]]; then
-	echo "max_connections is set to ${AUTO_CONNECTIONS}, which does not correlate with configmap value: ${cm_max_connections}"
+if [[ ${CONNECTIONS} != ${cm_max_connections} ]]; then
+	echo "with-limits: max_connections is set to ${CONNECTIONS}, which does not correlate with configmap value: ${cm_max_connections}"
 	exit 1
 fi
 
-apply_config "${test_dir}/conf/${cluster}2.yml"
+apply_config "${test_dir}/conf/${cluster}-with-requests.yml"
+wait_cluster_consistency "$cluster" 3
+
+if does_autotune_cm_exists; then
+	echo "with-requests: Operator shouldn't create autotune configmap just with resource requests"
```

Comment:
Should we exit here?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1275/commits/0305603ebc992dc4c27ed0e333e748122193b5f2